langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -202,6 +286,10 @@ public isolated function equalsIgnoreCaseAscii(string str1, string str2) returns
 #
 # The ASCII white space characters are 0x9...0xD, 0x20.
 #
+# ```ballerina
+# " BALLERINA  ".trim() ⇒ "BALLERINA"
```

Comment:
`io:println(" BALLERINA  ".trim())` wouldn't have the quotes?

Reply:
Good point. I actually wanted to chat on this. I have added all the string output within `""` . I felt it makes more sense if we differentiate and show explicitly that it is a string. Your thoughts?